src/dev/typescript/run_check_ts_projects_cli.ts

```diff
@@ -45,7 +45,7 @@ export async function runCheckTsProjectsCli() {
         }
 
         const file = new File(resolve(REPO_ROOT, line));
-        if (!file.isTypescript() || file.isFixture()) {
+        if (!file.isTypescript() || file.isTypescriptAmbient() || file.isFixture()) {
```

Comment:
I think we should check that `projects.length >= 1` for these files, rather than not checking them at all